utils/staking-miner/src/main.rs

```diff
@@ -322,7 +322,8 @@ async fn create_election_ext<T: EPM::Config, B: BlockT>(
 			},
 			..Default::default()
 		}))
-		.inject_hashed_prefix(&system_block_hash_key)
+		.inject_hashed_prefix(&<frame_system::BlockHash<T>>::prefix_hash())
+		.inject_hashed_key(&[twox_128(b"System"), twox_128(b"Number")].concat())
```

Comment:
This is needed, otherwise we can't create the extrinsic correctly -- the `fn additional_signed` of `CheckMortality` would fail.

Reply:
Does the module hash or module + storage item hash exist somewhere on the module itself? I think I recall Shawn mentioning something to that affect, but maybe you would need to import a runtime

Reply:
unfortunately `number` is not pub, so cannot be read this way... maybe with `StoargeInfo` it is possible though